### DISCUSSION TITLE
Relais module 3.3v

### DISCUSSION THREAD

**Discussion by alexdelli** [raomin/ESPAltherma#152] [Q&A] [answered]
Is this ok? 
![468AD75B-4F4A-4FB2-AD19-5F4B8884A6F4](https://user-images.githubusercontent.com/6674409/193088358-c040e161-9005-4429-b2d1-c121410dd2fa.jpeg)

i can’t find in Italy the original one in the how-to

**Comment by mdkeil** [comment, marked as answer]
yes, this one should work.
